package.json

```diff
@@ -77,7 +77,7 @@
   },
   "activationEvents": [
     "onView:mongoDB",
-    "onStartupFinished",
+    "onChatParticipant:mongodb.participant",
```

Comment:
Though it is best to be explicit, pretty sure `onView` and `onChatParticipant` get implicitly declared already. Removing `onStartupFinished`, I see no difference in performance, although the Macs may be too powerful to notice anything.
https://code.visualstudio.com/updates/v1_74#_implicit-activation-events-for-declared-extension-contributions